### PR TITLE
E2E tests for actor activation.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

E2E for actor activation and deactivation

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #820 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [X] Created/updated tests
* [X] Unit tests passing
* [X] End-to-end tests passing
* [ ] Extended the documentation
* [ ] Specification has been updated
* [ ] Provided sample for the feature

### CODE REVIEW

**Comment by artursouza** [comment]
There is a bug in dapr runtime. I will change the e2e test not to use UUID, so it will pass like it was before in my first local working version: https://github.com/dapr/dapr/issues/895

**Comment by artursouza** [comment]
> So is this test testing actor activation or actor deactivation? (according to the test cases I think it's mostly deactivation oriented?)

It tests both. There is a check in both tests to check is activation happened.

File: tests/e2e/actor_activation/actor_activation_test.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
+// +build e2e
+
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package actor_activation_e2e
```
Comment (artursouza) [inline tests/e2e/actor_activation/actor_activation_test.go]: I forgot to rename this, so I need another approval now.

File: tests/e2e/actor_activation/actor_activation_test.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+		_, err = utils.HTTPPost(invokeURL, []byte{})
+		require.NoError(t, err)
+
+		resp, err := utils.HTTPGet(logsURL)
+		require.NoError(t, err)
+		require.True(t, findActorActivation(resp, actorId))
+		require.True(t, findActorMethodInvokation(resp, actorId))
+		require.False(t, findActorDeactivation(resp, actorId))
+
+		time.Sleep(secondsToCheckActorDeactivation * time.Second)
```
Comment (LMWF) [inline tests/e2e/actor_activation/actor_activation_test.go]: Can you add a comment about how this flows?  Is it validating by asking the app for it's logs and parsing those logs?

Comment (LMWF) [reply tests/e2e/actor_activation/actor_activation_test.go]: Like what does find* mean?

Comment (artursouza) [reply tests/e2e/actor_activation/actor_activation_test.go]: Yes, it just scans the log entries. I can add a comment in my next PR.

**Pull request merged**